roles/kubernetes/master/templates/manifests/kube-controller-manager.manifest.j2

```diff
@@ -54,6 +54,8 @@ spec:
 {% endif %}
 {% if kube_network_plugin is defined and kube_network_plugin == 'cloud' %}
     - --configure-cloud-routes=true
+  {% else %}
```

Comment:
Remove spaces here

Reply:
Done in bcb879d0c54ea3211bc7d4f4c998e9efa61566d7